### PR TITLE
feat(apis): Assign API ownership - part 2

### PR DESCRIPTION
**Pull request by sentaur-athena**
Please review your teams ownership and comment if something is wrong. Teams covered in this PR:
Issues
Ingest
owners-native
feedback
hybrid cloud

### CODE REVIEW

File: src/sentry/api/endpoints/builtin_symbol_sources.py [hunk 1]
```diff
@@ -18,6 +19,7 @@ def normalize_symbol_source(key, source):
 
 @region_silo_endpoint
 class BuiltinSymbolSourcesEndpoint(Endpoint):
+    owner = ApiOwner.OWNERS_INGEST
```
Comment (Swatinem) [inline src/sentry/api/endpoints/builtin_symbol_sources.py:22]:
```suggestion
    owner = ApiOwner.OWNERS_NATIVE
```

Symbol sources should be part of native I believe

File: src/sentry/api/endpoints/event_apple_crash_report.py [hunk 2]
```diff
@@ -12,6 +13,7 @@
 
 @region_silo_endpoint
 class EventAppleCrashReportEndpoint(ProjectEndpoint):
+    owner = ApiOwner.OWNERS_INGEST
```
Comment (Swatinem) [inline src/sentry/api/endpoints/event_apple_crash_report.py:16]:
```suggestion
    owner = ApiOwner.OWNERS_NATIVE
```

Probably, though not quite sure?

File: src/sentry/api/endpoints/event_attachment_details.py [hunk 3]
```diff
@@ -48,6 +49,7 @@ def has_object_permission(self, request: Request, view, project):
 
 @region_silo_endpoint
 class EventAttachmentDetailsEndpoint(ProjectEndpoint):
+    owner = ApiOwner.OWNERS_INGEST
```
Comment (Swatinem) [inline src/sentry/api/endpoints/event_attachment_details.py:52]: Unless you introduce another option for "processing" team, I guess ingest would be the most appropriate

**Review by Swatinem — APPROVED** [review]

**Review by iker-barriocanal — COMMENTED** [review]
Left a comment in the endpoints assigned to but not owned by Ingest, the others LGTM.

File: src/sentry/api/endpoints/group_tombstone.py [hunk 4]
```diff
@@ -11,6 +12,7 @@
 
 @region_silo_endpoint
 class GroupTombstoneEndpoint(ProjectEndpoint):
+    owner = ApiOwner.HYBRID_CLOUD
```
Comment (markstory) [inline src/sentry/api/endpoints/group_tombstone.py:15]:
```suggestion
    owner = ApiOwner.ISSUES
```

group tombstones are part of discard and delete.

File: src/sentry/api/endpoints/group_tombstone_details.py [hunk 5]
```diff
@@ -11,6 +12,7 @@
 
 @region_silo_endpoint
 class GroupTombstoneDetailsEndpoint(ProjectEndpoint):
+    owner = ApiOwner.HYBRID_CLOUD
```
Comment (markstory) [inline src/sentry/api/endpoints/group_tombstone_details.py:15]:
```suggestion
    owner = ApiOwner.ISSUES
```

File: src/sentry/api/endpoints/organization_eventid.py [hunk 6]
```diff
@@ -14,6 +15,7 @@
 
 @region_silo_endpoint
 class EventIdLookupEndpoint(OrganizationEndpoint):
+    owner = ApiOwner.OWNERS_INGEST
```
Comment (markstory) [inline src/sentry/api/endpoints/organization_eventid.py:18]:
```suggestion
    owner = ApiOwner.ISSUES
```

This endpoint is used to find issues for event ids.

File: src/sentry/api/endpoints/organization_projects_sent_first_event.py [hunk 7]
```diff
@@ -9,6 +10,7 @@
 
 @region_silo_endpoint
 class OrganizationProjectsSentFirstEventEndpoint(OrganizationEndpoint):
+    owner = ApiOwner.OWNERS_INGEST
```
Comment (markstory) [inline src/sentry/api/endpoints/organization_projects_sent_first_event.py:13]:
```suggestion
    owner = ApiOwner.ISSUES
```

File: src/sentry/api/endpoints/project_event_details.py [hunk 8]
```diff
@@ -98,6 +99,7 @@ def get(self, request: Request, project, event_id) -> Response:
 
 @region_silo_endpoint
 class EventJsonEndpoint(ProjectEndpoint):
+    owner = ApiOwner.OWNERS_INGEST
```
Comment (markstory) [inline src/sentry/api/endpoints/project_event_details.py:102]:
```suggestion
    owner = ApiOwner.ISSUES
```

File: src/sentry/api/endpoints/project_create_sample_transaction.py [hunk 9]
```diff
@@ -67,6 +68,7 @@ def fix_event_data(data):
 
 @region_silo_endpoint
 class ProjectCreateSampleTransactionEndpoint(ProjectEndpoint):
+    owner = ApiOwner.OWNERS_INGEST
```
Comment (markstory) [inline src/sentry/api/endpoints/project_create_sample_transaction.py:71]:
```suggestion
    owner = ApiOwner.PERFORMANCE
```

File: src/sentry/api/endpoints/project_transaction_threshold.py [hunk 10]
```diff
@@ -42,6 +43,7 @@ def validate_threshold(self, threshold):
 
 @region_silo_endpoint
 class ProjectTransactionThresholdEndpoint(ProjectEndpoint):
+    owner = ApiOwner.OWNERS_INGEST
```
Comment (markstory) [inline src/sentry/api/endpoints/project_transaction_threshold.py:46]:
```suggestion
    owner = ApiOwner.PERFORMANCE
```

File: src/sentry/api/endpoints/event_attachment_details.py (hunk 3 above)
Comment (Swatinem) [reply src/sentry/api/endpoints/event_attachment_details.py:52]: TBH, I have no idea what the difference is. @ashwoods

Comment (jernejstrasner) [reply src/sentry/api/endpoints/event_attachment_details.py]: For Ingest we got rid of the separate owners team. I'll look into this for processing as well.
Please just use team-processing.

**Pull request merged**